### ISSUE TITLE
Clean npm install versions conflict

### ISSUE THREAD

**Issue by RRMoelker** [xebia/angular-form-messages-example#19]
npm install for a clean install request user version resolution.

```
Unable to find a suitable version for angular, please choose one:
    1) angular#1.4.9 which resolved to 1.4.9 and is required by angular-messages#1.4.9, angular-route#1.4.9
    2) angular#1.4 which resolved to 1.4.9 and is required by angular-form-messages-example
    3) angular#1.5.0 which resolved to 1.5.0 and is required by angular-mocks#1.5.0, angular-mocks#1.5.0
    4) angular#>=1.2 which resolved to 1.5.0 and is required by mox#0.4.3
```

**Comment by fvanwijk** [comment]
Yup I have to sync this example project with the newest version of angular-form-messages. I will also remove the bower stuff by using only npm dependencies

**Comment by RRMoelker** [comment]
For now consider PR #20

**Comment by fvanwijk** [comment]
`npm install` should work fine now (see Travis builds which are green again). Fixed by #22.
